app-typescript/components/TreeSelect/TreeSelect.tsx

```diff
@@ -42,6 +42,8 @@ interface IPropsBase<T> extends IInputWrapper {
     singleLevelSearch?: boolean;
     placeholder?: string;
     searchPlaceholder?: string;
+    noResultsFoundMessage?: string;
```

Comment:
make this one mandatory and do not provide a default value - it can't be translated.

Reply:
are u sure about mandatory?

Reply:
good point, many usages would break, let's do it another time